### ISSUE TITLE
[docs] outdated information on the website/blog

### ISSUE THREAD

**Issue by niieani** [rethinkdb/rethinkdb#6000]
Googling for `rethinkdb upsert` yields this page as the first result: https://www.rethinkdb.com/blog/answers-to-common-questions/#does-rethinkdb-support-upserts
But it seems the `upsert` option is no longer supported. The page should be updated to reflect the new equivalent way, which is `conflict: "update"`.

**Comment by danielmewes** [comment]
Thanks for reporting this @niieani .

There's a big note at the top of that blog post saying that it's outdated.

Still it seems like a lot of people keep finding it through search engines and don't read the note at the top of the post (I don't blame them, they are looking for a particular piece of information on the page after all).

@mglukhovsky Do you think it would make sense to exclude this blog post from search engines through a robot.txt?

**Comment by niieani** [comment]
You're right, I didn't notice the note, it's not really big, it's just long and italic ;)
If it were in big red letters, I'd probably notice it when skimming the page.

**Comment by VeXocide** [comment]
This is related to https://github.com/rethinkdb/docs/issues/1156.

**Comment by niieani** [comment]
One more idea: Since `upsert` is a general term known in other databases, people might be googling for exactly that. Removing the blog post from robots, or even altogether, doesn't really help users solve their problem. Perhaps it would be good to add a note about "how to do upserts" in the documentation page of the `insert` command API, actually mentioning the word `upsert`?

**Comment by danielmewes** [comment]
@niieani Good point.
Maybe instead of hiding it from search engines, we should actually consider editing the blog post and adding a note in the actual contents of it to point to the new syntax.

Independently of that, I think we should have an "upsert" recipe in our cookbook. There are already a bunch of examples in there that are essentially upserts, but with additional logic.
A plain "Insert document, or update fields if it already exists" recipe might be helpful, and we should make sure to use the term "upsert" to make it easier to discover from search engines.
https://www.rethinkdb.com/docs/cookbook/javascript/#manipulating-documents

@chipotle Could you take care of adding this to the cookbook?
